e2e/pages/events/manageOrders.page.js

```diff
@@ -69,7 +70,7 @@ const section3 = {
     },
   },
   childSelector: {
-    selector: index => `#childSelector_option${index}`,
+    selector: index => `#childSelector_option${index}-SELECTED`,
```

Comment:
bug: existing care order test didnt actually select the child from child selector block 'Which children?', just got lucky because theres only one child so if you continue without selecting, the code selects it in backend anyway